config/routes.rb

```diff
@@ -1,3 +1,3 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  root 'tests#home'
```

Comment:
I know I shouldn't use MVC, because view is unneeded.
But I haven't search the way to read the script and execute process automatically.
So first, I write it in MVC.